### DISCUSSION TITLE
能否支持超过2G的内容存入单key

### DISCUSSION THREAD

**Discussion by gxrsprite** [2881099/FreeRedis#216] [General]
redis自身是可以支持超过2g的文件的，设置配置文件即可
C#端由于byte[]数组的特性和托管内存的限制，导致不能超过2g大小
超过2g的需求来自于内网多服务架构，临时存大文件共享给其他服务，在内存够大的时候2g也不过是几百分之一，并不是大内存占用

**Comment by 2881099** [comment]
目前可以读：

redis.Get("key", stream);

写的话，因为 resp 协议问题，需要提前知道长度，可以考虑自己写 socket 发送命令。
